elkodon_pal/posix/build.rs

```diff
@@ -10,7 +10,7 @@ fn main() {
     #[cfg(any(target_os = "linux", target_os = "freebsd"))]
     println!("cargo:rustc-link-lib=pthread");
 
-    println!("cargo:rerun-if-changed=src/posix.h");
+    println!("cargo:rerun-if-changed=src/c/posix.h");
```

Comment:
Aaargh, it seems the build does not fail when the path is incorrect

Reply:
Argh ...